src/picasso-definition/scales/color/index.js

```diff
@@ -0,0 +1,21 @@
+import { hcl } from 'd3-color';
+
+export function makeDarker(c) {
```

Comment:
I suggest you move this file into `src/utils/color` and maybe call it `adjust-color.js` or something.

Reply:
Fixed